src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -88,7 +88,7 @@ const CollectorProfileSaves2Route: FC<CollectorProfileSaves2RouteProps> = ({
 
       <Spacer y={4} />
 
-      <Shelf showProgress={false}>
+      <Shelf showProgress={false} data-testid="artwork-list-rail">
```

Comment:
**question**: should we use `artwork-lists-rail` id here?

Reply:
Even better — I don't actually need this selector and the one in `ArtworkListItem.tsx` anymore. Thanks for reminding me. 

ae4d1fe831d9ff5efbde80e426c5fc05ae625627